programmers/두개-뽑아서-더하기.js

```diff
@@ -0,0 +1,16 @@
+function solution(numbers) {
+    const NUM_LENGTH = numbers.length;
+    let sum = 0;
+    let answer = [];
+    
+    for (let i = 0; i < NUM_LENGTH; i++) {
+        for (let j = i + 1; j < NUM_LENGTH; j++) { 
+            sum = numbers[i] + numbers[j];
+            if (answer.indexOf(sum) === -1){ 
+               answer.push(sum); 
+            }
```

Comment:
if문을 한 줄로 작성할 수 있으면 한 줄로 작성하기 그리고 indexOf대신 includes() 메서드를 사용하면 좀더 if문 조건식을 간결하게 작성할 수 있음
```suggestion
            if (!answer.includes(sum)) answer.push(sum);
```